apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/OperationsTest.java

```diff
@@ -138,7 +137,7 @@ private void runRetentionJobWithStringPartitionColumns(
   }
 
   @Test
-  public void testRetentionDoesNotCreateSnapshotsOnNoOpDelete() throws Exception {
+  public void testRetentionCreatesSnapshotsOnNoOpDelete() throws Exception {
```

Comment:
do we have a test to verify the delete statement generate a snapshot with operation != overwrite, and is that needed?

Reply:
Added